app/services/record_declaration.rb

```diff
@@ -205,6 +206,14 @@ def original_participant_declaration
     @original_participant_declaration ||= participant_declaration.duplicate_declarations.order(created_at: :asc).first
   end
 
+  def validate_not_retained_if_mentor
+    return unless declaration_type&.match?(/retained-\d/)
```

Comment:
Are we allowing extended declarations to be declared as well?

Reply:
I believe so, @cwrw can you confirm?

Reply:
ah good catch Leandro, we only allow started and completed declarations for mentors 2025, so everything else should raise an error

Reply:
I'll update the PR 👍